.github/workflows/test_editable_pip_install.yml

```diff
@@ -0,0 +1,63 @@
+name: Test Editable Pip Install
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  editable_install:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # This shouldn't be necessary, but for some reason it is on macos
+      - name: install dev_requirements.txt
+        if: ${{ matrix.os == 'macos-latest' }}
+        run: python3 -m pip install -r dev_requirements.txt
+
+      - name: Upgrade versions
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install setuptools==65.3.0
+
+      - name: editable install
+        run: python3 -m pip install -e .
+
+      - name: Set expected install locations for ubuntu
+        if: ${{ matrix.os == 'ubuntu-20.04' }}
+        run: |
+          echo CC_SYM_LOCATION=/home/runner/work/symforce/symforce/cc_sym.cpython-38-x86_64-linux-gnu.so >> $GITHUB_ENV
+          echo SYM_LOCATION=/home/runner/.local/lib/python3.8/site-packages/sym/__init__.py >> $GITHUB_ENV
+          echo SKYMARSHAL_LOCATION=/home/runner/.local/lib/python3.8/site-packages/skymarshal/__init__.py >> $GITHUB_ENV
+          echo SYMENGINE_LOCATION=/home/runner/work/symforce/symforce/third_party/symenginepy/symengine/__init__.py >> $GITHUB_ENV
+
+      - name: Set expected install locations for macos
+        if: ${{ matrix.os == 'macos-latest' }}
+        run: |
+          echo CC_SYM_LOCATION=/Users/runner/work/symforce/symforce/cc_sym.cpython-310-darwin.so >> $GITHUB_ENV
+          echo SYM_LOCATION=/usr/local/lib/python3.10/site-packages/sym/__init__.py >> $GITHUB_ENV
+          echo SKYMARSHAL_LOCATION=/usr/local/lib/python3.10/site-packages/skymarshal/__init__.py >> $GITHUB_ENV
+          echo SYMENGINE_LOCATION=/usr/local/lib/python3.10/site-packages/symengine/__init__.py >> $GITHUB_ENV
+
+      - name: Test cc_sym is installed in expected location
+        run: python3 -c "import cc_sym; assert cc_sym.__file__ == '$CC_SYM_LOCATION'"
+
+      - name: Test sym is installed in expected location
+        run: python3 -c "import sym; assert sym.__file__ == '$SYM_LOCATION'"
+
+      - name: Test skymarshal is installed in expected location
+        run: python3 -c "import skymarshal; assert skymarshal.__file__ == '$SKYMARSHAL_LOCATION'"
+
+      - name: Test symengine is installed in expected location
+        run: python3 -c "import symengine; assert symengine.__file__ == '$SYMENGINE_LOCATION'"
+
+      - name: Check that symengine is default SymForce backend
+        run: python3 -c 'import symforce; assert(symforce.get_symbolic_api() == "symengine")'
```

Comment:
Can we add a `import symforce.symbolic as sf; assert sf.sympy.__file__ == '$SYMENGINE_LOCATION'`?  `import symforce` isn't guaranteed to fully initialize everything (and even if it initializes all the relevant bits now it might not in the future)

Reply:
Okay. I think I've done what you meant.